sdk/eventhub/event-hubs/src/partitionContext.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * PartitionContext is passed into an EventProrcessor's initialization handler and contains information
+ * about the partition, the EventProcessor will be processing events from.
+ */
+export class PartitionContext {
```

Comment:
@chradek Do we need this to be a class?
Or are we making it a class to ensure that the properties are readonly?

Reply:
It should be safe to make this an interface now. If we want to ensure the properties are readonly at runtime we can either freeze the object that implements the interface `Object.freeze` or create the object using `Object.defineProperties`